x-pack/plugin/esql/qa/testFixtures/src/main/resources/floats.csv-spec

```diff
@@ -165,8 +165,8 @@ notLessThanMultivalue
 required_capability: mv_warn
 
 from employees | where not(salary_change < 1) | keep emp_no, salary_change | sort emp_no | limit 5;
-warning:Line 1:24: evaluation of [not(salary_change < 1)] failed, treating result as null. Only first 20 failures recorded.#[Emulated:Line 1:28: evaluation of [salary_change < 1] failed, treating result as null. Only first 20 failures recorded.]
```

Comment:
Indeed, with regex we have a way to manage these situations. 
As a follow-up I'll double-check and in case I'll remove it. In the meantime I opened this https://github.com/elastic/elasticsearch/issues/110092